modular_skyrat/modules/opposing_force/code/equipment/utility.dm

```diff
@@ -150,10 +150,38 @@
 			for rapid healing, a medical night vision HUD for quick identification of injured personnel, \
 			and other supplies helpful for a field medic."
 
+/datum/opposing_force_equipment/gear/ai_module
+	name = "Syndicate AI Law Module"
+	item_type = /obj/item/ai_module/syndicate
+	description = "When used with an upload console, this module allows you to upload priority laws to an artificial intelligence. \
+			Be careful with wording, as artificial intelligences may look for loopholes to exploit."
+
+/datum/opposing_force_equipment/gear/powersink
+	name = "Power Sink"
+	item_type = /obj/item/powersink
+	description = "When screwed to wiring attached to a power grid and activated, this large device lights up and places excessive \
+			load on the grid, causing a station-wide blackout. The sink is large and cannot be stored in most \
+			traditional bags and boxes. Caution: Will explode if the powernet contains sufficient amounts of energy."
+	admin_note = "Drains power from the station, explodes if overloaded."
+
+/datum/opposing_force_equipment/gear/powerbeacon
+	name = "Power Beacon"
+	item_type = /obj/item/sbeacondrop
+	description = "Not to be confused with the <i>Power Sink</i>.\
+			When screwed to wiring attached to an electric grid and activated, this large device pulls any \
+			active gravitational singularities or tesla balls towards it. This will not work when the engine is still \
+			in containment. Because of its size, it cannot be carried. Ordering this \
+			sends you a small beacon that will teleport the larger beacon to your location upon activation."
+	admin_note = "Generally requires a Supermatter Delamination to be effective."
+
 /datum/opposing_force_equipment/gear/cloak_mod
 	item_type = /obj/item/mod/module/stealth/ninja
 	description = "An upgraded MODsuit cloaking module stolen from the Spider Clan's finest. Consumes less power than the standard, but is obviously illegal."
 
+/datum/opposing_force_equipment/gear/flamethrower_mod
+	item_type = /obj/item/mod/module/chameleon
+	description = "A module that enables the user to disguise their MODsuit as any other type. Only works while undeployed."
```

Comment:
these dont match, I assume its just bc this is a draft PR and you're going to change this later, but important to have the fat reminder lest we cause jank